functions.ts

```
@@ -35,31 +37,33 @@ export const createNanoEventsWithWrappedEmit = emitHandler => ({
 });
 
 export function toBytes(value: number, byteCount: number) {
-  if (typeof value !== "number") {
-    logging.error("Invalid value type: " + value + " (" + typeof value + ")");
-    throw "InvalidValue";
+  if (typeof value !== 'number') {
+    logging.error('Invalid value type: ' + value + ' (' + typeof value + ')');
+    throw 'InvalidValue';
   }
 
   if (isNaN(value)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)